sklearn/manifold/mds.py

```diff
@@ -129,18 +131,18 @@ def _smacof_single(similarities, metric=True, n_components=2, init=None,
     return X, stress, it + 1
 
 
-def smacof(similarities, metric=True, n_components=2, init=None, n_init=8,
+def smacof(dissimilarities, metric=True, n_components=2, init=None, n_init=8,
```

Comment:
Maybe I'm missing something, but does it really require deprecation? It's not a keyword argument and the implementation hasn't changed, meaning it's backwards compatible. I was thinking of it more as a bugfix since passing an actual similarity/affinity matrix would've been incorrect.